libs/designsystem/src/lib/components/toast/services/toast.helper.spec.ts

```diff
@@ -78,11 +78,26 @@ describe('ToastHelper', () => {
           const ionToast = window.document.getElementsByTagName('ion-toast')[0];
           await TestHelper.whenReady(ionToast);
           const toastWrapper = ionToast.shadowRoot.querySelector('.toast-wrapper');
+          const expectedColor =
+            messageType !== 'danger' ? getColor(notificationColor) : getColor('warning');
```

Comment:
Consider readability:
```suggestion
          const expectedColor =
            messageType === 'danger' ? getColor('warning') : getColor(notificationColor);
```